nano/secure/ledger.hpp

```diff
@@ -116,6 +116,8 @@ class ledger final
 	// Returns the next receivable entry equal or greater than 'key'
 	std::optional<std::pair<nano::pending_key, nano::pending_info>> receivable_lower_bound (secure::transaction const & tx, nano::account const & account, nano::block_hash const & hash) const;
 	void initialize (nano::generate_cache_flags const &);
+
+public: // for unit tests only
```

Comment:
This is mainly used by confirming set, so the comment is inaccurate.

Reply:
This is still less than ideal. A safe pattern similar to the way it's done in other classes should be used that provides a public `force_confirm` method with a proper release_assert that guards it.

Reply:
Let's follow good patterns that are already established and work. A single assert is more descriptive than any comment.